src/lib/__test__/tradingviewUtils.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { ResolutionString } from 'public/tradingview/charting_library';
```

Comment:
love the tests!
i have a preference for test file names matching the file they are testing, to prevent overlap, duplication, or gaps in testing.
i'd recommend creating a `__test__` folder in the `lib/tradingView/dydxfeed` path, what do you think?

Reply:
yesss for sure